corehq/apps/accounting/models.py

```diff
@@ -779,7 +779,10 @@ class SoftwarePlan(models.Model):
         choices=SoftwarePlanVisibility.CHOICES,
     )
     last_modified = models.DateTimeField(auto_now=True)
-    is_customer_software_plan = models.BooleanField(default=False)
+    is_customer_software_plan = models.BooleanField(
+        default=False,
+        help_text="Determines if this plan can be used in a customer billing account",
```

Comment:
what was your reason for opting to add `help_text` and a migration versus adding a comment about this line?

Reply:
No good reason! Is there a reason to avoid small migrations like this?

Reply:
i felt like we went through a big effort of consolidating apps with tons of migrations a few years ago. I think for something like this I would err on not triggering a migration if that's the only change